clientgui/DlgAdvPreferencesBase.cpp

```diff
@@ -250,234 +279,335 @@ wxPanel* CDlgAdvPreferencesBase::createProcessorTab(wxNotebook* notebook)
+    wxStaticText* staticText24 = new wxStaticText(
+        box, ID_DEFAULT,
+        // context: 'In use' means mouse or keyboard input in last ___ minutes
+        _("'In use' means mouse or keyboard input in last"),
+        wxDefaultPosition, wxDefaultSize, 0
     );
-    m_chkProcOnBatteries->SetToolTip(
-        _("Check this to suspend computing on portables when running on battery power.")
+    m_txtProcIdleFor = new wxTextCtrl(
+        box, ID_TXTPROCIDLEFOR, wxEmptyString, wxDefaultPosition, getTextCtrlSize(wxT("999.99")), wxTE_RIGHT
+    );
+    wxStaticText* staticText25 = new wxStaticText(
+        box, ID_DEFAULT,
+        // context: 'In use' means mouse or keyboard input in last ___ minutes
+        _("minutes"),
+        wxDefaultPosition, wxDefaultSize, 0
     );
-    suspendComputingBoxSizer->Add( m_chkProcOnBatteries, 0, wxALL, 5 );
+    addNewRowToSizer(box_sizer, ProcIdleForTT, staticText24, m_txtProcIdleFor, staticText25);
 
+    // suspend while in use
+    //
     m_chkProcInUse = new wxCheckBox(
-        suspendComputingStaticBox, ID_CHKPROCINUSE,
+        box, ID_CHKPROCINUSE,
         _("Suspend when computer is in use"), wxDefaultPosition, wxDefaultSize, 0
     );
     m_chkProcInUse->SetToolTip(
         _("Check this to suspend computing and file transfers when you're using the computer.")
     );
-    suspendComputingBoxSizer->Add( m_chkProcInUse, 0, wxALL, 5 );
+    box_sizer->Add( m_chkProcInUse, 0, wxALL, 5 );
 
+    // suspend GPU while in use
+    //
     m_chkGPUProcInUse = new wxCheckBox(
-        suspendComputingStaticBox, ID_CHKGPUPROCINUSE,
+        box, ID_CHKGPUPROCINUSE,
         _("Suspend GPU computing when computer is in use"), wxDefaultPosition, wxDefaultSize, 0
     );
     m_chkGPUProcInUse->SetToolTip(
         _("Check this to suspend GPU computing when you're using the computer.")
     );
-    suspendComputingBoxSizer->Add( m_chkGPUProcInUse, 0, wxALL, 5 );
+    box_sizer->Add( m_chkGPUProcInUse, 0, wxALL, 5 );
 
-    // min idle time
-    wxString ProcIdleForTT(_("This determines when the computer is considered 'in use'."));
-
-    wxStaticText* staticText24 = new wxStaticText(
-            suspendComputingStaticBox, ID_DEFAULT,
-            // context: 'In use' means mouse/keyboard input in last ___ minutes
-            _("'In use' means mouse/keyboard input in last"),
-            wxDefaultPosition, wxDefaultSize, 0
-        );
-
-    m_txtProcIdleFor = new wxTextCtrl(
-        suspendComputingStaticBox, ID_TXTPROCIDLEFOR, wxEmptyString, wxDefaultPosition, getTextCtrlSize(wxT("999.99")), wxTE_RIGHT
+    // max # CPUs and throttling
+    //
+    wxStaticText* staticText20 = new wxStaticText(
+        box, ID_DEFAULT, _("Use at most"), wxDefaultPosition, wxDefaultSize, 0
+    );
+    m_txtProcUseProcessors = new wxTextCtrl(box, ID_TXTPROCUSEPROCESSORS, wxEmptyString, wxDefaultPosition, textCtrlSize, wxTE_RIGHT );
+    wxStaticText* staticText22 = new wxStaticText(
```

Comment:
Precede line 336 with `/*xgettext:no-c-format*/`. For an explanation why this is needed, see [here](https://stackoverflow.com/questions/70490036/xgettext-incorrectly-infers-c-format-in-cpp-files) and [here](https://www.mit.edu/afs.new/athena/astaff/source/src-9.4/third/gettext2/gettext-tools/doc/gettext_3.html#SEC18).